File: generated-sources/kotlin-spring/mojang-authentication/src/main/kotlin/com/github/asyncmc/mojang/authentication/kotlin/spring/model/ReducedUserData.kt
package com.github.asyncmc.mojang.authentication.kotlin.spring.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import com.github.asyncmc.mojang.authentication.kotlin.spring.model.GameProfileProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param id 
 * @param properties 
 */
data class ReducedUserData (

        @JsonProperty("id") val id: String? = null,

        @JsonProperty("properties") val properties: List<GameProfileProperty>? = null
) {

}

File: generated-sources/scala-gatling/mojang-authentication/src/gatling/scala/com/github/asyncmc/mojang/authentication/scala/gatling/model/ReducedUserData.scala

package com.github.asyncmc.mojang.authentication.scala.gatling.model


case class ReducedUserData (
    _id: Option[String],
    _properties: Option[List[GameProfileProperty]]
)
object ReducedUserData {
    def toStringBody(var_id: Object, var_properties: Object) =
        s"""
        | {
        | "id":$var_id,"properties":$var_properties
        | }
        """.stripMargin
}
